### PR TITLE
Generation of obs_meta_timespan

### PR DESCRIPTION
**Pull request by antoniocarlon**
Generation of obs_meta_timespan using OBS_Timespan

Closes https://github.com/CartoDB/bigmetadata/issues/420
Related to https://github.com/CartoDB/bigmetadata/issues/351

### CODE REVIEW

**Comment by antoniocarlon** [comment]
Updated. CR round 2?

File: tasks/carto.py [hunk 1]
```diff
@@ -799,25 +835,97 @@ class OBSMeta(Task):
...
+timespan_tags, timespan_extra, timespan_type, timespan_aggregate, numers, denoms, geoms, the_geom, timespan_version)
+SELECT denom_timespan::TEXT timespan_id,
+       denom_timespan_alias::TEXT timespan_alias,
+       denom_timespan_name::TEXT timespan_name,
+       denom_timespan_description::TEXT timespan_description,
+       denom_timespan_range::DATERANGE timespan_range,
+       denom_timespan_weight::NUMERIC timespan_weight,
+       NULL::JSONB timespan_tags, --FIRST(timespan_tags)::JSONB timespan_tags,
+       NULL::JSONB timespan_extra,
+       NULL::TEXT timespan_type,
```
Comment (ethervoid) [inline tasks/carto.py]: Why aren't we storing the type?

File: tasks/carto.py [hunk 2]
```diff
@@ -799,25 +835,97 @@ class OBSMeta(Task):
...
+       NULL::Geometry(Geometry, 4326) the_geom, -- ST_Union(DISTINCT ST_SetSRID(the_geom, 4326)) the_geom
+       NULL::Integer timespan_version
+FROM observatory.obs_meta_next
+WHERE geom_timespan IS NOT NULL
+GROUP BY geom_timespan, geom_timespan_alias, geom_timespan_name,
+         geom_timespan_description, geom_timespan_range, geom_timespan_weight
+ON CONFLICT (timespan_id) DO NOTHING;
+        ''',
+        '''
+UPDATE {obs_meta} AS t SET numers =
```
Comment (ethervoid) [inline tasks/carto.py]: We have to discuss that because I don't think is necessary

File: tasks/carto.py (hunk 1 above)
Comment (ethervoid) [reply tasks/carto.py]: We're storing timespan for all the types not segregated by type so we cant set the type yet ( backward compabtility)

**Comment by ethervoid** [comment]
Blocked until we have #412 merged

**Pull request merged**